### PR TITLE
Add an option to ignore restrictpkg

### PR DESCRIPTION
**Pull request by zoetrope**

### CODE REVIEW

**Review by kfyharukz — APPROVED** [review]
~~LGTM~~

File: golang-bionic/analyzer/pkg/restrictpkg/testdata/src/ignore/ignore.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package a
```
Comment (kfyharukz) [inline golang-bionic/analyzer/pkg/restrictpkg/testdata/src/ignore/ignore.go:1]: The package name should be `ignore`, shouldn't it?

File: golang/analyzer/pkg/restrictpkg/testdata/src/ignore/ignore.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package a
```
Comment (kfyharukz) [inline golang/analyzer/pkg/restrictpkg/testdata/src/ignore/ignore.go:1]: ditto

**Review by kfyharukz — CHANGES_REQUESTED** [review]

File: golang-bionic/analyzer/pkg/restrictpkg/testdata/src/ignore/ignore.go (hunk 1 above)
Comment (zoetrope) [reply golang-bionic/analyzer/pkg/restrictpkg/testdata/src/ignore/ignore.go:1]: I think it should be `main`. Because it has a main function.

**Review by kfyharukz — APPROVED** [review]
LGTM

**Pull request merged**